docs/development.md

```diff
@@ -96,12 +94,10 @@ current context in your kubeconfig.
 gcloud config set project <PROJECT_ID>
 # Ensure kubectl context is set to correct cluster
 kubectl config set-context <CONTEXT>
-# Build images
-make build-images
-# Push images to GCR
-make push-images
+# Build images/manifests and push images
+make config-sync-manifest
 # Run the tests with image prefix/tag from previous step and desired test regex
```

Comment:
Maybe add instructions here about how to test with a different image tag?

Reply:
Done, PTAL